installer/targets/cmdgen/install.bash

```diff
@@ -6,7 +6,7 @@ then
     echo "Installing GPSRCmdGen to ~/src/GPSRCmdGen"
 
     echo "Installing dependency of GPSRCmdGen: mono-complete"
-    sudo apt-get install mono-complete
```

Comment:
Why not use:
tue-install-system-now

Reply:
Merging for now. Changes can applied when other PR is merged.